tests/Functional/Form/PHPCRTypeGuesserTest.php

```diff
@@ -266,7 +266,7 @@ public function testReference()
 
         $formBuilder
             ->add('single', null, ['class' => ReferrerDocument::class])
-            ->add('documents', null, ['class' => ReferrerDocument::class])
+            ->add('documents', null, ['class' => TestDocument::class])
```

Comment:
this is unrelated to the persistence library change, right? but looks like a good idea, so lets keep it.

Reply:
Yeah, it was a bit of mess because I tried first to make changes for the persistence library, but then after fixing some warnings this was the build failing https://travis-ci.org/github/doctrine/DoctrinePHPCRBundle/builds/732697516